admin/jobs/river/autoscaler_jobs.go

```diff
@@ -42,13 +58,13 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 	for _, rec := range recs {
 		targetProject, err := w.admin.DB.FindProject(ctx, rec.ProjectID)
 		if err != nil {
-			w.logger.Debug("failed to find project", zap.String("project_name", targetProject.Name), zap.Error(err))
+			w.logger.Debug("failed to find project", zap.String("project_name", rec.ProjectID), zap.Error(err))
```

Comment:
It's weird that the field is called `project_name`, but now reports an ID.

Honestly I get a little scared seeing diffs inside the jobs logic. I only caught it here because Git tracked it as a file rename, but most of the other jobs have been tracked as new lines of code, making it very difficult to review if a change happened or not.

Can you:
1. Refactor this PR such that each file in `admin/worker` maps 1:1 to a new file in `admin/jobs/river`? Such that Git can report correct diffs so we can validate that the jobs are not changed.
2. Validate that no changes are made to the job logic? If you want to change anything in the job logic, we should do it in a separate PR after.